e2e/cloudflare-workers/cloudflare-workers.e2e.ts

```
@@ -78,18 +78,18 @@ describe.skipIf(gatewayRunner !== 'node')('Cloudflare Workers', () => {
-        'OTLP_SERVICE_NAME:' + env.OTLP_SERVICE_NAME,
-        ...(isDebug() ? ['--var', 'DEBUG:1'] : []),
-      ],
-    });
+    await spawn([
+      'yarn',
+      'wrangler',
+      'dev',
+      '--port',
+      port.toString(),
+      '--var',
+      'OTLP_EXPORTER_URL:' + env.OTLP_EXPORTER_URL,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)